lib/active_storage/storage_services.yml

```diff
@@ -13,6 +13,7 @@ amazon:
   secret_access_key: <%= Rails.application.secrets.aws[:secret_access_key] %>
   region: us-east-1
   bucket: your_own_bucket
+  endpoint: s3.amazonaws.com
```

Comment:
Let’s omit this option in the default config since it doesn’t need to be specified unless you’re using an S3-compatible service. We can document it at some point.